test/library/standard/List/perf/listBenchmark1.chpl

```diff
@@ -0,0 +1,172 @@
+//
+// A performance test benchmarking common list operations.
+//
+
+private use HaltWrappers;
+private use List;
+private use Random;
+private use Time;
+
+type byte = int(8);
+type testList = list(byte, false);
+
+config param isPerformanceTest: bool = false;
+config const n0: int = 50000;
+
+const n1: int = n0 / 1000;
+const seed = 314159;
+const rand = createRandomStream(eltType=int, seed=seed, parSafe=false);
+
+proc createList(size: int) {
+  var result: testList;
+  for i in 1..size do result.append(0:byte);
+  return result;
+}
+
+proc generateNoise() {
```

Comment:
What's the noise? Is this really just to warmup? If so, I'd probably just call it `warmup`

Reply:
Sure. Wasn't sure what to call it.